test/makeChange_test.js

```diff
@@ -28,7 +28,7 @@ describe('makeChange()', function(){
     })
   })
 
```

Comment:
I recommend adding the following to make your tests more through.  Right now all of your tests have a price of 100.
```
    expect(makeChange({price: 159, amountGiven: 200})).to.deep.equal({
      quarters: 1,
      dimes: 1,
      nickels: 1,
      pennies: 1,
    })

    expect(makeChange({price: 432, amountGiven: 500})).to.deep.equal({
      quarters: 2,
      dimes: 1,
      nickels: 1,
      pennies: 3,
    })
```